### ISSUE TITLE
Redirecting process ?

### ISSUE THREAD

**Issue by Youssef1313** [microsoft/calculator#731]
With a code like this:

```vb
Dim ProcID As Integer
' Start the Calculator application, and store the process id.
ProcID = Shell("CALC.EXE", AppWinStyle.NormalFocus)
' Activate the Calculator application.
AppActivate(ProcID)
```

The `ProcID` returned by `Shell` isn't correct. Is the process some kind of process redirection or something like that ?
That was causing a problem in a sample in the dotnet documentations.
For more information, see https://github.com/dotnet/docs/issues/7668

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by HowardWolosky** [comment]
Thanks for the question @Youssef1313, as well as for the pointer to dotnet/docs#7668.  I've added some feedback there as well.

`calc.exe` does not contain the Calculator application in Windows 10.  It continues to exist on the system for backwards compatibility reasons, and all that it does is launch the Store version of calculator (the application hosted in this project).  This has been the design within Windows for many years now.

Given that they are two completely different processes (the `calc.exe` "launcher" and `calculator.exe` itself), there is no way for this specific example to work.  Closing this Issue out as there is no additional work that can be done on our end, and I've added some thoughts to dotnet/docs#7668 to ensure that they use a different example.
